my-react-app/src/store/reducer/formSlice.ts

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import { FormInput } from 'models/models';
+
+const formState: FormInput[] = [];
```

Comment:
You should use an object with properties.
```suggestion
const formState = {
  submittedForms: [] as FormInput[],
};
```